src/server/common_services/auth_server.js

```diff
@@ -591,10 +591,6 @@ function _prepare_auth_request(req) {
         dbg.log3('load auth system:', req.system && req.system._id);
     };
 
-    req.has_bucket_permission = function(bucket) {
```

Comment:
has_s3_bucket_permission uses this function - not sure you can just remove. Also not sure what is our plan about this once allowed buckets are removed? to deny all requests or to allow them.